components/expander.jsx

```diff
@@ -16,7 +16,7 @@ var Expander = React.createClass({
       expanded: true
     });
   },
-  onClick: function() {
+  handleClick: function(e) {
```

Comment:
Good idea--I've noticed this is a convention they use in the React tutorial and I've tried to stick to it myself.